programming/Encoders.md

```diff
@@ -0,0 +1,57 @@
+<b>Ever been on a website written in another language and had to utilize google translate to gain some 
+semblance of recognition as to what it was you were looking at?</b>
+Encoders work in a similar fashion to Google Translate, of course instead of translating languages it merely translates
+given data into another format.
+
+There's lots of different examples of encoders at work all around us, such as:
+  * Elevators - Encoders in elevators tell the elevator's controller device its position so that it can properly align itself with the floor and open up the doors
+  at the correct position.
+  * Cranes - Some large cranes use encoders in order to collect accurate location information to know where it is.
+  * Computer Microphones - Encoders are used to convert those signals your microphone picks up into actual data that your computer can understand!
+  
+Encoders are the reason our robot can use a camera for visual recognition or record it's speed, or know when it's hit a brick wall.
+
+<b>TL;DR:  Encoders are the duct tape which binds our data together.</b>
+
+<b>How do Encoders work?</b>
```

Comment:
Another minor formatting nit ... changing this line to be formatted as: `*How do Encoders works?*` will still be bold, but a bit easier to read.